Makefile

```diff
@@ -35,7 +41,9 @@ check-formatting::
 	done
 
 validate-examples::
```

Comment:
I think it'd be useful to also (try and) execute `valuesets/source-update/update-rat.sh` in the Makefile, and as a dependent task for the task `validate-valuesets`.

Reply:
was already suggested and left out of scope
summary: chesterton's fence (@ryanbnl  ;)

Reply:
> and as a dependent task for the task `validate-valuesets`.

an absolute no

Reply:
I think I see your point. Would it be useful to document the existence and function of `update-rat.sh` in the README? (If yes: I'm happy to do that.)

Reply:
Is primarily meant as a tool for whoever is managing the governance of the central deployment - so do make sure you mention that in the README